types/component/AnonymousChat/BaseChatItem.types.ts

```diff
@@ -21,9 +21,10 @@ export interface BaseChatItemProps {
 }
 
 export interface ChatItemMyTextProps extends BaseChatItemProps {
-  type?: BaseChatItemTypeProps.MY_ANON_CHAT;
-  status: ChatStatus;
+  status?: ChatStatus;
   AnonymousImage: React.ReactElement;
+  chatType?: string;
+  avatar?: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `status` property in the `ChatItemMyTextProps` interface has been made optional, and two new properties `chatType` and `avatar` have been added. Ensure that all usages of this interface are updated to handle the possibility of `status` being undefined and to utilize the new `chatType` and `avatar` properties if necessary.
```diff
-  status: ChatStatus;
+  status?: ChatStatus;
+  chatType?: string;
+  avatar?: string;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->